frontend/src/scenes/experiments/ExperimentView/components.tsx

```diff
@@ -263,18 +269,31 @@ export function ActionBanner(): JSX.Element {
         )
     }
 
-    // Trend: running, results present, not significant
-    if (isExperimentRunning && experimentResults && !isExperimentStopped && !areResultsSignificant) {
-        return (
-            <LemonBanner type="info" className="mt-4">
-                Your experiment is live and is collecting data, but hasn't yet reached the statistical significance
-                needed to make reliable decisions. It's important to wait for more data to avoid premature conclusions.
-            </LemonBanner>
-        )
-    }
-
     // Running, results present, not significant
     if (isExperimentRunning && experimentResults && !isExperimentStopped && !areResultsSignificant) {
+        // Results insignificant, but a large enough sample/running time has been achieved
+        // Further collection unlikely to change the result -> recommmend cutting the losses
+        if (
+            experimentInsightType === InsightType.FUNNELS &&
+            funnelResultsPersonsTotal > Math.min(recommendedSampleSize, 500)
```

Comment:
We agreed on this condition when we discussed this last week. But shouldn't this be `max()`? i.e. being conservative and discouraging the user only after the larger of the two values has been met?

Reply:
yes yes

Reply:
To this condition, how about we add a time guard as well? Imagine this is a high volume funnel, so 500 people enter during the first day and recommendedSampleSize is broken (at 0). It seems weird to already say 'cut your losses'.

Maybe we should wait for at least 2 days anyway?

Reply:
yep, will add 👍